### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by anybodesign** [matthieua/WOW#320]
Hi,
Using CSS `visibility: hidden` is causing an accessibility issue. Hidden elements are not accessible to keyboard navigation and screen reading equipment. Could be replaced by `opacity: 0`.
:)

**Comment by lsterling03** [comment]
I have this issue as well. Screen readers skip over content that has visibility hidden.
